src/commands/scorecard.js

```diff
@@ -0,0 +1,109 @@
+// Import Node.js Dependencies
+import fs from "node:fs";
+
+// Import Third-party Dependencies
+import cliui from "@topcli/cliui";
+import kleur from "kleur";
+import * as scorecard from "@nodesecure/ossf-scorecard-sdk";
+import ini from "ini";
+
+// VARS
+const { yellow, grey, cyan, white } = kleur;
+
+function separatorLine() {
+  return grey("-".repeat(80));
+}
+
+export function getCurrentRepository() {
+  const result = {
+    ok: true,
+    reason: null,
+    value: null
+  };
+  // eslint-disable-next-line no-sync
+  const config = ini.parse(fs.readFileSync(".git/config", "utf-8"));
+
+  const originMetadata = config["remote \"origin\""];
+  if (!originMetadata) {
+    result.ok = false;
+    result.reason = "Cannot find origin remote.";
```

Comment:
Why not just returning an object each time? instead of mutating result (just make implementation harder to read for me).

Reply:
Fixed with `@openally/result`